azure/activity_logs_monitoring/index.js

```diff
@@ -225,39 +225,58 @@ class EventhubLogForwarder {
         return this.addTagsToJsonLog(jsonLog);
     }
 
-    extractResourceId(record) {
+    formatSourceType(sourceType) {
+        if (sourceType.includes('microsoft.')) {
+            return sourceType.replace('microsoft.', 'azure.');
+        } else {
+            return '';
+        }
+    }
+
+    extractMetadataFromResource(record) {
         var metadata = { tags: [], source: '' };
         if (
             record.resourceId === undefined ||
             typeof record.resourceId !== 'string'
         ) {
             return metadata;
-        } else if (
-            record.resourceId.toLowerCase().startsWith('/subscriptions/')
-        ) {
-            var resourceId = record.resourceId.toLowerCase().split('/');
-            if (resourceId.length > 2) {
-                metadata.tags.push('subscription_id:' + resourceId[2]);
+        }
+        var resourceId = record.resourceId.toLowerCase().split('/');
+        if (resourceId[0] === '') {
```

Comment:
I'm not sure what format the resource id's are, but this will have unexpected behavior if the string has the form `"// multiple slashes"`.

Reply:
We really only expect the resource to come in like starting with a slash, and splitting by slash will give us an empty string as the first item. I made it conditional in case for some reason azure sends us something without a slash but I don't think that's likely, and i don't think it would be likely that they'd send something with multiple slashes.